src/main/java/org/irods/Util/Permission.java

```diff
@@ -7,8 +7,16 @@ public enum Permission {
     NULL("null"),
     READ("read"),
     WRITE("write"),
-    OWN("own");
-    
+    OWN("own"),
+    DELETE_OBJECT("delete_object"),
+    CREATE_OBJECT("create_object"),
+    DELETE_METADATA("delete_metadata"),
+    MODIFY_METADATA("modify_metadata"),
+    CREATE_METADATA("create_metadata"),
+    READ_METADATA("read_metadata"),
+    MODIFY_OBJECT("modify_object"),
+    READ_OBJECT("read_object");
```

Comment:
Please group these with their aliases respectively.

Reply:
please keep `own` on top and `read_object` at the bottom... with the aliases in there/alongside like kory suggested.

that ordering is most easy to parse later when the next reader comes along

Reply:
sorry, read_object is not at the bottom... please put null at the bottom.   and the rest in decreasing order (they are linear).

https://docs.irods.org/4.3.2/icommands/user/#ichmod

Reply:
ok, so just in the same order as that doc you linked?

Reply:
Done.

Reply:
yes, thanks.